src/main/java/uk/gov/hmcts/reform/roleassignment/launchdarkly/FeatureToggleService.java

```diff
@@ -50,7 +66,31 @@ public boolean isValidFlag(String flagName) {
         return ldClient.isFlagKnown(flagName);
     }
 
-    public Map<String, String> getLaunchDarklyMap() {
-        return launchDarklyMap;
+    public String getLaunchDarklyFlag(HttpServletRequest request) {
+        String uri = request.getRequestURI();
+        switch (request.getMethod()) {
+            case GET:
+                if (getRequestMap.get(uri) != null) {
+                    return getRequestMap.get(uri);
+                } else if (uri.contains(URI_GET_ASSIGNMENTS_BY_ACTOR_ID)) {
+                    return LD_FLAG_GET_ROLE_ASSIGNMENTS_BY_ACTOR_ID;
+                }
```

Comment:
we can put the 'URI_GET_ASSIGNMENTS_BY_ACTOR_ID' path into the same map and avoid this else part same for delete.

Reply:
Hi Aashish, as discussed, we cannot put it into map as it is a wild card match, i.e.  "String.contains(url)" .